lib/condorExe.py

```diff
@@ -1,9 +1,8 @@
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
-# Description:
-#   This module implements the functions to execute condor commands
-
+"""This module implements the functions to execute condor commands.
+"""
```

Comment:
Should this be added at the end of the previous line itself? (looked at other docstrings in this file for reference before adding this comment)

Reply:
done